packages/appium/docs/en/ecosystem/build-drivers.md

```diff
@@ -581,22 +581,30 @@ call something like:
 driver.executeScript('soundz: playSong', [{song: 'Stairway to Heaven', artist: 'Led Zeppelin'}]);
 ```
 
-Then in your driver code you could simply look for `soundz:` at the beginning of your
-`execute` override, and do the appropriate thing:
+Then in your driver code you could simply define the magic member field `this.executeOverloads` as
+a mapping of script names to member functions on your driver. All you need to do at that point is
+implement the `execute` the command to take advantage of the built-in `executeOverload` method
+available on all drivers:
 
 ```js
-async execute(script, args) {
-  if (script === 'soundz: playSong') {
-    const {song, artist} = args[0];
-    // play the song based on song and artist details
-    return;
-  }
+executeOverloads = {'soundz: playSong': this.soundzPlaySong}
+
+async soundzPlaySong({song, artist}) {
+  const {song, artist} = args[0];
```

Comment:
where args is defined here?

Reply:
copy/paste error.

Reply:
it still there. Where the changes not pushed?